src/components/Question.jsx

```diff
@@ -313,42 +312,44 @@ export default class Question extends React.Component {
     );
   }
 
-  static renderQuestionHelp(question, options) {
-    const icons = options.icons;
-    let questionHelpIcon;
-
-    for (let i = 0; i < icons.length; i++) {
-      if (icons[i].id === Constants.ICONS.QUESTION_HELP) {
-        questionHelpIcon = icons[i];
-      }
-    }
-
-    if (!questionHelpIcon || questionHelpIcon.behavior === Constants.ICON_BEHAVIOR.ENABLE) {
-      if (question[Constants.HELP_DESCRIPTION]) {
+  static evaluateIcon(icon, question, options, onCommentChange) {
```

Comment:
@LaChope 

not sure what  `evaluateIcon` means ? in what way do you evaluate ? In what sense it is evaluation ?

how about ?
`static getIconComponent(iconId, question, options, onCommentChange)`